deps.edn

```diff
@@ -1,18 +1,18 @@
 {:paths ["src/main" "resources"]
  :deps
  {;; Clojure core deps
-  org.clojure/clojure                      {:mvn/version "1.10.3"}
+  org.clojure/clojure                      {:mvn/version "1.11.1"}
   org.clojure/tools.logging                {:mvn/version "1.1.0"}
   org.clojure/core.memoize                 {:mvn/version "1.0.250"}
   clojure-interop/java.security            {:mvn/version "1.0.5"}
   ;; Util deps
   ;; - java-time: Updating to 0.3.3 causes dep clash with DATASIM
   camel-snake-kebab/camel-snake-kebab      {:mvn/version "0.4.2"}
   cheshire/cheshire                        {:mvn/version "5.11.0"}
-  clojure.java-time/clojure.java-time      {:mvn/version "0.3.2"}
+  clojure.java-time/clojure.java-time      {:mvn/version "1.2.0"}
```

Comment:
Is there an ns change that comes along with this one?

Reply:
Actually yes - `java.time` has become `java.time.api`. IDK why any tests haven't failed though

Reply:
We can also kill the `java-time: Updating to 0.3.3 causes dep clash with DATASIM` comment too